readme.txt

```diff
@@ -105,105 +105,90 @@ There are opportunities for developers at all levels to contribute. [Learn more
 
 = 4.7 =
 
-* Release date:
-* Release post:
-
-**New Features and Improvements**
+* Release date: March 6th, 2017
+* Release post: http://wp.me/p1moTy-46L
 
 **Performance Improvements**
 
-* Development mode: improve performance by only calling `site_url()` once. #6130
-* Media Summary: improve performance with single page load caching. #5938
-* Open Graph: Avoid slow queries on sites with thousands of images. #6297
-* Widgets: update the Top Posts Widget to retrieve posts via the WordPress.com REST API. #6335
-* Widget Visibility: Rewrite the way that the major/minor rules lists are generated to save bandwidth and memory. #6333
+* Experience better performance with single page load caching of the media summary.
+* We made some improvements to avoid slow queries on sites with thousands of images.
+* The Top Posts widget now utilizes an endpoint from the WP.com REST API improving performance, and update flexibility.
+* Improve development mode performance by only calling `site_url()` once.
+* We rewrote the way major/minor rules lists are generated to save bandwidth, and memory for sites with lots of widgets.

```

Comment:
Let's mention Widget Visibility? 

`We improved Widget Visibility performance to save bandwidth and memory for sites with lots of widgets.`?